src/pages/home/ReportScreen.js

```diff
@@ -212,51 +204,55 @@ class ReportScreen extends React.Component {
         if (isArchivedRoom) {
             reportClosedAction = lodashFindLast(this.props.reportActions, action => action.actionName === CONST.REPORT.ACTIONS.TYPE.CLOSED);
         }
-
         return (
             <ScreenWrapper style={[styles.appContent, styles.flex1, {marginTop: this.state.viewportOffsetTop}]}>
                 <KeyboardAvoidingView>
                     <HeaderView
                         reportID={reportID}
                         onNavigationMenuButtonClicked={() => Navigation.navigate(ROUTES.HOME)}
                     />
-
                     <View
                         nativeID={CONST.REPORT.DROP_NATIVE_ID}
                         style={[styles.flex1, styles.justifyContentEnd, styles.overflowHidden]}
+                        onLayout={event => this.setState({skeletonViewContainerHeight: event.nativeEvent.layout.height})}
```

Comment:
NAB, so guessing this is not dynamic since we just set the container height once?

Reply:
yeah thats right.